app/views/evidence/_confirmation_part_remission.html.slim

```diff
@@ -1,8 +1,14 @@
 .row
   .small-12.medium-12.large-12.columns
     .evidence-confirmation-letter
-      =t('evidence.part-remission.letter_html',
+      =t('letters.part-payment.send_part_payment_html',
          full_name: confirmation.application.full_name,
-         current_user: current_user.name,
-         fee: confirmation.amount_to_pay,
-         date: confirmation.expires_at.strftime(Date::DATE_FORMATS[:gov_uk_long]))
+         reference: confirmation.application.reference,
+         user_name: current_user.name,
+         part_payment_amount: "£#{confirmation.amount_to_pay}",
+         total_income: "£#{confirmation.income}",
+         children: confirmation.application.children,
+         date_for_part_payment: confirmation.expires_at.strftime(Date::DATE_FORMATS[:gov_uk_long]),
+         name_delivery_manager: 'YOUR MANAGERS NAME',
+         court_tribunal_name: 'APPEAL COURT NAME',
+         date_for_appeal: 'INSERT DATE TO APPEAL')
```

Comment:
@leighmoney I set the text for these three fields to address my comment below...